alphastats/plots/VolcanoPlot.py

```diff
@@ -357,7 +357,9 @@ def _annotate_result_df(self):
         add color labels for up and down regulates
         """
         self.res = self.res[(self.res["log2fc"] < 20) & (self.res["log2fc"] > -20)]
-        self.res["-log10(p-value)"] = -np.log10(self.res[self.pvalue_column])
+        self.res["-log10(p-value)"] = [
```

Comment:
I don't quite get the change. Isn't it the same, but slower?

Reply:
With the changes to the upstream functions, masked p-values emerge (automatic completeness filtering). Since self.res["-log10(p-value)"] is a pandas Series, it tries to use the pandas log10 function even though we write np.log10. The pandas function can't handle the masked values though (log10 0 error) and the only way I found to handle them was to iterate manually.

Reply:
I added a todo comment.